cluster-operator/src/main/java/io/strimzi/operator/cluster/model/AbstractModel.java

```diff
@@ -103,8 +102,7 @@
     public static final String ENV_VAR_DYNAMIC_HEAP_MAX = "DYNAMIC_HEAP_MAX";
     public static final String NETWORK_POLICY_KEY_SUFFIX = "-network-policy";
 
-    private static final String DELETE_CLAIM_ANNOTATION =
-            ClusterOperator.STRIMZI_CLUSTER_OPERATOR_DOMAIN + "/delete-claim";
+    private static final String DELETE_CLAIM_ANNOTATION = Annotations.STRIMZI_DOMAIN + "/delete-claim";
```

Comment:
Since we are already touching all these places, should we use the same naming schema? Most of these variables seem to use `ANNO_*`.